backend/.python-version

```diff
@@ -1 +1 @@
-3.9.0
+3.9.6
```

Comment:
Is there any specific reason to set the version to 3.9.6 here? The python version we use in our Dockerfile is actually [3.9.13](https://github.com/docker-library/python/blob/6ca54f5e8a180fad4b525c2dfecd23c76ff04894/3.9/slim-bullseye/Dockerfile#L27C22-L27C26)

Reply:
@chandrasekharan-zipstack for now, since we see local versions for 3.9.6 (@muhammad-ali-e  & @gaya3-zipstack ), we are sticking to this. This is more of a trial and run pinning. If there are more developers using 3.9.13, then we can pin it to that and other devs need to roll to 3.9.13. Basically @hari-kuriakose mentioned that this is a forcible way to standardise our developer settings.

Reply:
@chandrasekharan-zipstack We are explicitly setting it to `3.9.6` in local to make it consistent across all dev systems. It won't affect other places as it's still `3.9.x`.